packages/datatrak-web/src/features/Survey/SurveyContext/SurveyContext.tsx

```diff
@@ -26,6 +26,7 @@ const defaultContext = {
   numberOfScreens: 0,
   screenNumber: 1,
   screenHeader: '',
+  screenDetail: '',
```

Comment:
My inclination would’ve been to default to `undefined`, but it seems there’s a pattern here of using some sort of defined "empty" value?

```suggestion
  screenDetail: undefined,
```